src/hubverse_infrastructure/shared/hubverse_transforms.py

```diff
@@ -210,16 +210,19 @@ def create_lambda_package_placeholder(s3_bucket: str, s3_key: str):
         raise Exception(f"Error when checking for existing lambda package: {s3_bucket}/{s3_key}") from e
 
 
-def create_transform_infrastructure():
+def create_transform_infrastructure() -> tuple[aws.lambda_.Function, aws.iam.Role]:
     """
     Create all AWS infrastructure required to support the lambda function that will
     operate on cloud-based model-output files.
     """
     bucket_name = "hubverse-assets"
     lambda_name = "hubverse-transform-model-output"
     lambda_package_location = "s3://hubverse-assets/lambda/hubverse-transform-model-output.zip"
-    lambda_package_path = CloudPath(lambda_package_location)
+    lambda_package_path = CloudPath(lambda_package_location)  # type: ignore
```

Comment:
the type checker doesn't seem to grok `CloudPath`, so tell it to ignore this line